packages/actions/README.md

```diff
@@ -6,17 +6,17 @@
 </p>
 
 <p align="center">
-    <a href="https://github.com/privacy-scaling-explorations/p0tion">
+    <a href="https://github.com/ethstorage/p0tion">
```

Comment:
Why can't we reuse p0tion's action package?

Reply:
we need to publish our package, as the config needs to change.